grab.go

```diff
@@ -21,5 +22,12 @@ func Grab(sc types.StackerConfig, storage types.Storage, name string, source str
 	}
 	defer os.Remove(path.Join(sc.RootFSDir, name, "rootfs", "stacker"))
 
+	if len(hash) > 0 {
+		if err = c.Execute(fmt.Sprintf("echo %s %s | sha256sum --check", hash, source), nil); err != nil {
```

Comment:
This seems to not work properly. Am I missing something here?

Reply:
Yes -- Execute() just uses execv() ultimately, and thus these things are not running inside of a shell, so using shell syntax like `|` here will not work.

Additionally, it would be good if we can avoid executing stuff inside the container if at all possible. Can we hash this on the host after we've imported it? I believe this will keep all the import hashing code in one place as well.

Reply:
Understood, I will hash it on the host. Thank you for the detailed answer.